planner/core/testdata/plan_suite_in.json

```diff
@@ -1186,5 +1186,21 @@
       "select a, count(*) from t group by a -- shouldn't be rewritten",
       "select sum(a) from t -- sum shouldn't be rewritten"
     ]
+  },
+  {
+    "name": "TestRemoveRedundantPredicates",
+    "cases":[
+      "select f from t use index() where f = 1 and f = 1 -- simple redundancy of exact condition",
```

Comment:
`use index()` hint means we will use the `tableScan` not use the `indexScan`. Why we use this hint in here?